### ISSUE TITLE
[Package Issue]: Package.Identifer

### ISSUE THREAD

**Issue by mrhvid** [microsoft/winget-pkgs#70219]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installation issue.

### Brief description of your issue

`winget upgrade Jabra.Direct`    returns

Installer hash does not match; to override this check use --force

### Steps to reproduce

`winget upgrade Jabra.Direct`
Found Jabra Direct [Jabra.Direct] Version 6.2.18901
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licenses to, third-party packages.
Downloading https://jabraxpressonlineprdstor.blob.core.windows.net/jdo/JabraDirectSetup.exe
  ██████████████████████████████  86.8 MB / 86.8 MB
**Installer hash does not match; to override this check use --force**

### Actual behavior

The installer hash doesn't match the manifest

### Expected behavior

The hash is expected to match the manifest

### Environment

```shell
winget --info
Windows Package Manager v1.3.2091
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1889
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.18.2091.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir
```

**Comment by mdanish-kh** [comment]
Will be resolved when #70055 gets merged

**Comment by mrhvid** [comment]
cool thank you :)
